doc/getting-started/apl-nuc.rst

```diff
@@ -133,6 +140,165 @@ complete this setup.
       |                    | which is enterprise-style kernel with backports   |
       +--------------------+---------------------------------------------------+
 
+.. _quick-setup-guide:
```

Comment:
Placing this section here doesn't make sense.  Do I need to run the steps immediately before this (I don't' think so), and the steps immediately after this scripted setup aren't separated enough.   I was wanting the steps to first describe the automated/scripted path, and THEN you could follow them with a clear line and a heading for the manual installation method.  As it's written now, the automated/scripted steps are in between some of the manual steps and it looks confusing.

Reply:
Re-placing "quick-setup-guide" and add a head line for the rest step of clear linux os installation.